modules/administration-guide/pages/repairing-the-devworkspace-operator.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: PROCEDURE
+:navtitle: Repairing the {devworkspace} Operator
+:description: Repairing the {devworkspace} Operator
+:keywords: devworkspace, operator, repair
+:page-aliases:
+
+[id="repairing-the-devworkspace-operator_{context}"]
+= Repairing the {devworkspace} Operator
+
+.Prerequisites
+
+* An active `{orch-cli}` session as a cluster administrator to the destination {orch-name} cluster. See {orch-cli-link}.
+
```

Comment:
Missing a prerequisite explaining when it may work: are the version of {prod-short} or {devworkspace} operator relevant?

Reply:
Like the rest of the docs content, it's for the product version and devworkspace operator version that are specified in antora.yml.